ui/src/App.tsx

```diff
@@ -131,9 +154,14 @@ const App: React.FC = () => {
           onExecute={executePython}
           onPlayToEnd={onPlayToEnd}
           executing={current.value == 'idle'}
+          error={pythonError}
+        />
+      </div>
+      <div className={styles.panel}>
+        <Console
+          stdout={current.context.data?.[current.context.index]?.stdout || ''}
```

Comment:
debugger-fsm defines a helper method `currentTrace` that would be helpful here:
``` js
currentTrace(current.context?)?.stdout ?? ''
```

Reply:
Fixed in latest commit!